week-1/implement/sum.js

```diff
@@ -0,0 +1,21 @@
+function sum(array) {
+    // armazenar a soma
+    let total = 0;
+  
+    
+    for (const element of array) {
+      // Verifique se o elemento é um número
+      if (typeof element === 'number') {
+        // Adicione o número à soma total
+        total += element;
+      }
+    }
+  
+    
+    return total;
+  }
```

Comment:
Again, a nice and clean implementation for your solution function, Great job, Laura. One thing I would like to say is, you can definitely name your function better.